<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaNuevaContrasena</class>
 <widget class="QDialog" name="VentanaNuevaContrasena">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIDEPASS</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="input_cuenta"/>
    </item>
    <item>
     <widget class="QLineEdit" name="input_detalle"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>401</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_cancelar">
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_generar">
      <property name="font">
       <font>
        <family>Berlin Sans FB</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Generar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Ejemplo, (Facebook)</string>
   </property>
   <property name="text">
    <string>Ingresa nombre de la cuenta:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Berlin Sans FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Ejemplo cuenta de facebook juanitoperez@gmail.com</string>
   </property>
   <property name="text">
    <string>Ingresa el detalle:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
